.golangci.yml

```diff
@@ -56,9 +61,6 @@ linters-settings:
     rules:
       - name: exported
         severity: warning
-  ifshort:
```

Comment:
this removal is not necessary.

Reply:
`ifshort` has been deprecated and is considered buggy apparently. And since it's removed above, I suspect, that this setting no longer makes sense?

Reply:
Oh i missed to see ifshort in disabled linters list, thanks 😄